csharp/ql/test/library-tests/csharp8/NullCoalescingAssignment.ql

```diff
@@ -0,0 +1,7 @@
+import csharp
+
+query predicate nullcoalescing(NullCoalescingExpr expr) { any() }
+
+query predicate assignments(AssignCoalesceExpr expr, Expr expanded) {
+  expanded = expr.getExpandedAssignment()
```

Comment:
Really nice how this "just works", and that the CFG gets the correct `null`/non-`null` successors for the second access to `o` in the expanded assignment `o = o ?? this`.